packages/cmf/__tests__/route.test.js

```diff
@@ -44,6 +44,22 @@ describe('loadComponent behavior', () => {
 		expect(mockItem.component.displayName).toBe('WithView');
 	});
 
+	it('should use the componentId', () => {
```

Comment:
`should use the componentId` is a to vague statement to describe what the CMF route do here.